Doppler.Sap/Extensions/SapMappersServiceCollectionExtension.cs

```diff
@@ -0,0 +1,22 @@
+using Doppler.Sap.Factory;
+using Doppler.Sap.Mappers.BusinessPartner;
+using Microsoft.Extensions.DependencyInjection;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Doppler.Sap.Extensions
```

Comment:
Para este tipo de extensiones se suele usar el namespase _Microsoft.Extensions.DependencyInjection_, eso ayuda a reducir los usins en _Startup_.

```suggestion
namespace Microsoft.Extensions.DependencyInjection
```

Reply:
Done